data/json/items/magazine/exodii.json

```
@@ -37,5 +37,24 @@
     "reload_time": 180,
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "123ln": 60 } } ]
+  },
+  {
+    "id": "exodiisapramag5",
+    "looks_like": "exodiiminimag",
+    "type": "MAGAZINE",
+    "name": { "str": "Sapra '36 magazine" },
+    "description": "A large, 5 round single stack magazine for the Sapra '36.",
```

```suggestion
    "description": "A large, five round single stack magazine for the Sapra '36.",
```
Proper gramma' demands <10 is spelt